src/Core/Grid/Definition/Factory/CartRuleGridDefinitionFactory.php

```diff
@@ -0,0 +1,262 @@
+use PrestaShop\PrestaShop\Core\Grid\Column\Type\Common\ToggleColumn;
+use PrestaShop\PrestaShop\Core\Grid\Column\Type\DataColumn;
+use PrestaShop\PrestaShop\Core\Grid\Filter\Filter;
+use PrestaShop\PrestaShop\Core\Grid\Filter\FilterCollection;
+use PrestaShop\PrestaShop\Core\Hook\HookDispatcherInterface;
+use PrestaShopBundle\Form\Admin\Type\DateRangeType;
+use PrestaShopBundle\Form\Admin\Type\SearchAndResetType;
+use PrestaShopBundle\Form\Admin\Type\YesAndNoChoiceType;
+use Symfony\Component\Form\Extension\Core\Type\TextType;
+
+/**
+ * Class responsible for providing columns, filters, actions for cart price rule list.
+ */
+final class CartRuleGridDefinitionFactory extends AbstractGridDefinitionFactory
+{
+    const GRID_ID = 'cart_rule';
+
+    public function __construct(
+        HookDispatcherInterface $hookDispatcher
+    ) {
+        parent::__construct($hookDispatcher);
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function getId(): string
+    {
+        return self::GRID_ID;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function getName(): string
+    {
+        return $this->trans('Cart rules', [], 'Admin.Catalog.Feature');
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function getColumns(): ColumnCollection
+    {
+        return (new ColumnCollection())
+            ->add((new BulkActionColumn('bulk'))
+            ->setOptions([
+                'bulk_field' => 'id_cart_rule',
+            ])
+            )
+            ->add((new DataColumn('id_cart_rule'))
+            ->setName($this->trans('ID', [], 'Admin.Global'))
+            ->setOptions([
+                'field' => 'id_cart_rule',
+            ])
+            )
+            ->add((new DataColumn('name'))
+            ->setName($this->trans('Name', [], 'Admin.Global'))
+            ->setOptions([
+                'field' => 'name',
+            ])
+            )
+            ->add((new DataColumn('priority'))
+            ->setName($this->trans('Priority', [], 'Admin.Global'))
+            ->setOptions([
+                'field' => 'priority',
+            ])
+            )
+            ->add((new DataColumn('code'))
+            ->setName($this->trans('Code', [], 'Admin.Global'))
+            ->setOptions([
+                'field' => 'code',
+            ])
+            )
+            ->add((new DataColumn('quantity'))
+            ->setName($this->trans('Quantity', [], 'Admin.Catalog.Feature'))
+            ->setOptions([
+                'field' => 'quantity',
+            ])
+            )
+            ->add((new DateTimeColumn('date_to'))
+            ->setName($this->trans('Expiration date', [], 'Admin.Catalog.Feature'))
+            ->setOptions([
+                'format' => 'Y-m-d H:i',
+                'field' => 'date_to',
+            ])
+            )
+            ->add((new ToggleColumn('active'))
+            ->setName($this->trans('Enabled', [], 'Admin.Global'))
```

Comment:
We definitely need to harmonize this wording; for the whole Catalog part of the back office, we have three different wordings for this column:

- in Products it is 'Status'
- in Categories it is 'Displayed'
- in Monitoring it is 'Status'
- in Brand & Suppliers it is 'Enabled'
- in Discounts it is 'Status'
- in Stocks it is 'Status'

I recommend to use _Status_ since error and success notifications when activating the item mention the word 'status'.

Reply:
I've just created an issue (#19142) to have it investigated a bit and then harmonized by the Content Manager. :-)